### PR TITLE
Refactor step trends functions

### LINKED ISSUE #2: Activity.js positive/negative step trends duplicate functionality
- getNegativeStepTrends() and getPositiveStepTrends() might be able to be consolidated into a single function.

### PR DESCRIPTION
**Pull request by cammac60**
- This PR combines the `getPositiveStepTrends` and `getNegativeStepTrends` functions into a single function (`getStepTrends`) and updates the tests/DOM usage.

---------------

- The new function begins on line 88 of ActivityRepository.js.
- Changes to the DOM are on lines 280/281 of index.js.
- Changes to testing are on lines 145 and 149 of Activity-test.js.

---------------

- `npm run start` to make sure the elements still appear correctly on the DOM.
- `npm test` to make sure all tests are passing.

---------------

This PR resolves #2 

@brittanystoroz @khalidwilliams

### CODE REVIEW

File: src/ActivityRepository.js [hunk 1]
```diff
@@ -85,31 +85,33 @@ class ActivityRepository {
     return this.user.slice(index - 6, index + 1);
   }
 
-  getPositiveStepTrends() {
-    return this.user.reduce((acc, day, index) => {
-      if (index < 2) {
+  getStepTrends(type) {
```
Comment (brittanydionigi) [inline src/ActivityRepository.js]: A better parameter name here might be `trendDirection` - type is a little vague

File: src/ActivityRepository.js [hunk 2]
```diff
@@ -85,31 +85,33 @@ class ActivityRepository {
...
         return acc;
+        }, []);
       }
-      if ((day.numSteps < this.user[index - 1].numSteps) &&
-        (this.user[index - 1].numSteps < this.user[index - 2].numSteps)) {
-        acc.push(day.date);
-      }
-      return acc;
-    }, []);
+    }
```
Comment (brittanydionigi) [inline src/ActivityRepository.js:112]: As we kind of saw yesterday, there's not a great way to DRY up this repetitive code as it only differs based on the greater than/less than operator. The only solution I could think of, which is likely even HAIRIER is doing something like:

```js
if type === positive
    comparisonOperator = '>';
else 
    comparisonOperator = '<'
```

then in your conditional logic you would swap out the greater than / less than signs for `eval(comparisonOperator)`

This would let you use a variable to replace the hardcoded greater than / less than signs. But you need to use this weird `eval` function that tells JavaScript "treat the string thats passed in as some sort of JavaScript that actually needs to be evaluated"

Long story short, dont do this. Just leave that code as-is for now despite its ugliness,

File: src/ActivityRepository.js (hunk 1 above)
Comment (cammac60) [reply src/ActivityRepository.js]: Thanks for the suggestion, Brittany! I renamed that and made another push.

File: src/ActivityRepository.js (hunk 2 above)
Comment (cammac60) [reply src/ActivityRepository.js:112]: @trishalanglois @allisonjw Per Brittany's comment, do you think we should close this PR and shelf the refactoring on this for now?
